avr-hal-generic/src/wdt.rs

```diff
@@ -0,0 +1,141 @@
+//! WDT Implementation
+
+pub use embedded_hal::watchdog;
```

Comment:
I think this import is unused?

Reply:
I think this is what allows the wdt module in the chip to bring the watchdog traits into scope.

```
//! WatchDog Timer Implementation

pub use avr_hal_generic::wdt::watchdog::*;
pub use avr_hal_generic::wdt::*;
```

Reply:
Do we still need them?  The macro is explicitly referring to the full path so I'd say the import is unnecessary.  If downstream code wants to name them, it should be preferred to import `embedded-hal` directly.